mappings/net/minecraft/world/World.mapping

```diff
@@ -80,30 +318,192 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 soundType
+		ARG 5 pitch
+	METHOD method_225 isFireOrLavaInBox (Lnet/minecraft/class_25;)Z
+		ARG 1 box
+	METHOD method_226 unloadEntitySkin (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_227 tickEntities ()V
+	METHOD method_228 getTopSolidBlockY (II)I
+		ARG 1 x
+		ARG 2 z
 	METHOD method_229 setBlock (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-		ARG 4 id
-	METHOD method_235 (IIII)V
+		ARG 4 blockId
+	METHOD method_230 worldEvent (IIIII)V
+		ARG 1 event
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 data
+	METHOD method_231 remove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_232 doLightingUpdates ()Z
+	METHOD method_233 getSpawnPositionValidityY (II)I
+		ARG 1 x
+		ARG 2 z
+	METHOD method_234 isAir (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_235 blockUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_236 serverRemove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_237 updateSkyBrightness ()V
+	METHOD method_238 hasChunk (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_239 isPosLoaded (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_240 setBlocksDirty (IIII)V
+		ARG 1 x
+		ARG 2 z
+		ARG 3 minY
+		ARG 4 maxY
+	METHOD method_241 updateEntity (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_242 tick ()V
+	METHOD method_243 blockUpdateEvent (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_244 notifyNeighbors (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_245 updateWeatherCycles ()V
+	METHOD method_246 setBlockDirty (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_247 camTransferPowerInDirection (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
+	METHOD method_248 manageChunkUpdatesAndEvents ()V
+	METHOD method_249 canPlaceAt (III)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-	METHOD method_243 (III)V
+	METHOD method_250 isEmittingRedstonePowerInDirection (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-	METHOD method_253 (IIII)V
+		ARG 4 direction
+	METHOD method_251 saveDimensionData ()V
+	METHOD method_252 getBrightness (III)I
 		ARG 1 x
-		ARG 2 t
+		ARG 2 y
+		ARG 3 z
+	METHOD method_253 neighborUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
 		ARG 3 z
+		ARG 4 blockId
 	METHOD method_254 getSeed ()J
+	METHOD method_255 getLightLevel (III)I
 	METHOD method_256 getTime ()J
+	METHOD method_257 isTopY (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
 	METHOD method_258 getSpawnPos ()Lnet/minecraft/class_63;
+	METHOD method_259 getChunkManager ()Lnet/minecraft/class_51;
+	METHOD method_260 removeBlockEntity (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_261 getDimensionData ()Lnet/minecraft/class_52;
+	METHOD method_262 getProperties ()Lnet/minecraft/class_7;
+	METHOD method_263 canTransferPower (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_264 updateSleepingPlayers ()V
+	METHOD method_265 isEmittingRedstonePower (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_266 skipNight ()V
```

Comment:
This function just wakes up all the players and clears weather, it doesn't really change world time at all. Could this be renamed to `onNightFinished` or something. In MCP the name is `wakeUpAllPlayers`

Reply:
I think we decided on call that `afterSkipNight` would be a good name for it

Reply:
`afterSkipNight` sounds good